6001-7000/LIT6011FeastsList.xml

```diff
@@ -36,9 +36,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Chronography"/>
-                    <term key="Liturgy"/>
```

Comment:
See above, on Chronography. I would leave "Liturgy" (also for another List above), this depends on the understanding of the keyword, of course